lzy/allocator/src/main/java/ai/lzy/allocator/alloc/DeleteSessionAction.java

```diff
@@ -130,4 +119,34 @@ private StepResult exec() {
 
         return StepResult.RESTART;
     }
+
+    private StepResult removeNetPolicy() {
+        try {
+            allocationContext.networkPolicyManager().deleteNetworkPolicy(sessionId);
+        } catch (Exception e) {
+            if (e instanceof KubernetesClientException && !isNotRetryable((KubernetesClientException) e)) {
```

Comment:
This action must not know about k8s i think
Lets throw some abstract or custom exception here

Reply:
I would rather make multiple catch blocks or just one block with KubernetesClientException (because we don't do anything with not-k8s exception). I agree, that abstract exception is more acceptable, but in this case we won't go away from k8s. And it's already done in mount/unmount actions, so we should use similar approaches or find more elegant solution

Reply:
And I would add log message before this 'if block' to make sure we log every exception